rest-api/dao/participant_dao.py

```diff
@@ -344,6 +344,34 @@ def add_missing_hpo_from_site(self, session, participant_id, site_id):
     # Update the version and add history row
     self._do_update(session, participant, participant)
 
+  def switch_to_test_account(self, session, participant_id):
+    test_hpo = HPODao().get_by_name(TEST_HPO_NAME)
+    if test_hpo is None:
+      raise BadRequest('No test hpo found for name ' + TEST_HPO_NAME)
+
+    test_hpo_id = test_hpo.hpoId
+    site = self.site_dao.get_by_hpo_id(test_hpo_id)
+    if site is None:
+      raise BadRequest('Invalid siteId reference hpo_id %r.' % test_hpo_id)
+
+    participant = self.get_for_update(session, participant_id)
+    if participant is None:
+      raise BadRequest('No participant %r for HPO ID udpate.' % participant_id)
+
+    if participant.hpoId == test_hpo_id:
+      return
+
+    participant.hpoId = site.hpoId
+    participant.organizationId = site.organizationId
+    participant.siteId = site.siteId
+    participant.providerLink = make_primary_provider_link_for_id(site.hpoId)
```

Comment:
using providerLink is deprecated (we should put a note in _do_update). Also, using provider link will prevent some of the auto pairing based on site|org|hpo. It's a use one or the other situation, but not both.[https://github.com/all-of-us/raw-data-repository/blob/master/rest-api/dao/participant_dao.py#L152](url)